Purchases/Public/RCPurchases.m

```diff
@@ -359,11 +359,11 @@ - (void)postAttributionData:(NSDictionary *)data
         RCErrorLog(@"⚠️ The parameter networkUserId is REQUIRED for AppsFlyer. ⚠️");
     }
     NSString *networkKey = [NSString stringWithFormat:@"%ld",(long)network];
-    NSString *advertisingIdentifier = [self.attributionFetcher advertisingIdentifier];
+    NSString *identifierForAdvertisers = [self.attributionFetcher identifierForAdvertisers];
```

Comment:
I don't think `identifierForAdvertisers` will trigger any rejections right? Any other safer idea

Reply:
I tested in the simulator that this value is the same when running the code in develop. If someone can double test it it would be great.

Reply:
sorry for the delay, I'll test it out as well

Reply:
works correctly on my device, same value before & after the PR

Reply:
if we get rejections with this name, we'll refactor to `theThingyForAdvertisers`